psutil/_psutil_windows.c

```diff
@@ -2694,9 +2694,13 @@ static char *get_region_protection_string(ULONG protection) {
  */
 static PyObject *
 psutil_proc_memory_maps(PyObject *self, PyObject *args) {
+	#ifdef _WIN64
+		MEMORY_BASIC_INFORMATION64 basicInfo;
+	#else
+		MEMORY_BASIC_INFORMATION basicInfo;
+	#endif
```

Comment:
Please indent the `ifdef`s properly (no spaces) and also the variable names should be indented like the others below